lib/caiquenator/phrases.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Caiquenator.Phrases do
+  @external_resource "priv/data.yml"
+  @phrases File.cwd!()
+           |> Path.join(@external_resource)
+           |> YamlElixir.read_from_file!()
+
+  def get_random do
+    Enum.random(@phrases)
+  end
```

Comment:
Since this is a very small function, sometimes we can write it in a just single line. Like: 
```elixir
def get_random, do: Enum.random(@phrases)
```
wdyt?